src/app.ts

```diff
@@ -1,13 +1,30 @@
 import express from 'express'
 import path from 'path'
+import cookieParser from 'cookie-parser'
 const app = express()
 const port = process.env.PORT || 3000
+const adminRouter = express.Router()
```

Comment:
create a new folder called `admin`. Then create a file inside `admin` called `index.ts`. Then put this router stuff inside that file and export it. Let's not clutter the main file.